Models/User.js

```diff
@@ -0,0 +1,35 @@
+const { Schema, model } = require('mongoose');
+///////////////////////////////
+// MODELS
+////////////////////////////////
+const VideoidSchema = new Schema(
+  {
+  URL: String,
+  log: String,
+  },
+    { timestamps: true }
+);
+
+const ObjectidSchema = new Schema(
+  {
+  Time: String,
+  Geoposition: String,
+  isActive: Boolean,
+  video: [Videoid],
+  },
+    { timestamps: true }
+);
+
+const UserSchema = new Schema(
+  {
+  userid: String,
+  name: String,
+  email: String,
+  log: [Objectid],
+  },
+    { timestamps: true }
+);
+
+const User = model("User", UserSchema);
```

Comment:
we will have to define a model for not only our User but as well for Video, and Log. 

`
const Video = model("Video", VideoSchema)
const Log = ...
`